### ISSUE TITLE
Tooltip component traps focus indefinitely

### ISSUE THREAD

**Issue by brunnerh** [carbon-design-system/carbon-components-svelte#924]
Depending on the browser, the `Tooltip` component can prevent tabbing back or forward from it.

On Chrome one cannot tab to the focusable element before the Tooltip (Shift + Tab), on Firefox one cannot tab to the focusable element after.

Example:

```svelte
<TextInput labelText="Input Before"/>
<Tooltip triggerText="Tooltip">
	<TooltipFooter>
		<div/>
		<Button>
			Tab Capture
		</Button>
	</TooltipFooter>
</Tooltip>
<TextInput labelText="Input After"/>
```

[REPL](https://svelte.dev/repl/479900acaa6d4dac932328dd5dcea3e2?version=3.44.2)

**Comment by metonym** [comment]
Using the React implementation as a reference, tabbing/shift+tabbing continues to trap the focus. However, pressing the "Escape" key will close the tooltip and focus the icon, allowing one to escape the focus and continue shift+tabbing.

**Comment by metonym** [comment]
Fixed in [v0.52.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.52.0)
